adafruit_mcp230xx/mcp23s08.py

```diff
@@ -0,0 +1,91 @@
+# SPDX-FileCopyrightText: 2017 Tony DiCola for Adafruit Industries
+# SPDX-FileCopyrightText: 2019 Carter Nelson
+#
+# SPDX-License-Identifier: MIT
+
+"""
+`MCP23S08`
+====================================================
+
+CircuitPython module for the MCP23S08 I2C I/O extenders.
+
+* Author(s): Tony DiCola
+"""
+
+from micropython import const
+from .MCP23Sxx import MCP23SXX
+from .digital_inout import DigitalInOut
+
+__version__ = "0.0.0-auto.0"
+__repo__ = "https://github.com/adafruit/Adafruit_CircuitPython_MCP23Sxx.git"
+
+_MCP23S08_ADDRESS = const(0x20)
+_MCP23S08_IODIR = const(0x00)
+_MCP23S08_IPOL = const(0x01)
+_MCP23S08_GPINTEN = const(0x02)
+_MCP23S08_DEFVAL = const(0x03)
+_MCP23S08_INTCON = const(0x04)
+_MCP23S08_IOCON = const(0x05)
+_MCP23S08_GPPU = const(0x06)
+_MCP23S08_INTF = const(0x07)
+_MCP23S08_INTCAP = const(0x08)
+_MCP23S08_GPIO = const(0x09)
```

Comment:
There are missing property accessors for some these registers when compared with mcp23s17.py

Reply:
Yes because the 08 variant doesn't have the 2nd bank so it doesn't need b side variables, this is the same as the MCP23008 implementation but with the SPI device bus instead.